packages/server/src/events/event.entity.ts

```diff
@@ -2,25 +2,25 @@ import { Table, Model, Column, DataType } from 'sequelize-typescript'
 
 @Table({ tableName: 'events' })
```

Comment:
I was thinking we leave them out for now since we're not really using the `events` table in the MVP and it's easy to add retroactively — but I'm happy to add the indices if you think it's a good idea to future-proof.